src/libs/actions/PersonalDetails.js

```diff
@@ -262,6 +262,39 @@ function setPersonalDetails(details, shouldGrowl) {
         });
 }
 
+function updateProfile(firstName, lastName, pronouns, timezone) {
+    const myPersonalDetails = personalDetails[currentUserEmail];
+    API.write('UpdateProfile', {
+        details: JSON.stringify({firstName, lastName, pronouns}),
```

Comment:
This would require changes on the Web side too, but it would be better to have these parameters separated. That way, we'll have better `_inputRules` checking on each of them (I just got this comment on my PR 😆)

Reply:
Good idea! Made an issue for this https://github.com/Expensify/Expensify/issues/220321

Hopefully we can get this merged quicker (once I address other comments) then I'll come back to this refactor 👍

Reply:
Adding `firstName`, `lastName`, `pronouns` below so my next Web-E PR can accept those & not care about `details` anymore